website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -483,8 +483,9 @@ variables is shown in the table below.
 The orca module provides the outputs seen below in the table. Outputs are
 accessed using terraform dot syntax in the format of `module.orca.variable_name`.
 
-| Output Variable                               | Description                               |
-| --------------------------------------------- | ----------------------------------------- |
+| Output Variable                                      | Description                               |
+| -----------------------------------------------------|---------------------------------------------------------|
+| `orca_catalog_reporting_api_invoke_url`              |The URL to invoke the API for catalog reporting lambda |
```

Comment:
See comments from modules/lambdas/main.tf. I am thinking we should just have a single base URL instead of three.